velox/docs/functions/spark/datetime.rst

```diff
@@ -9,4 +9,26 @@ These functions support TIMESTAMP and DATE input types.
 
 .. spark:function:: year(x) -> integer
 
-    Returns the year from ``x``.
\ No newline at end of file
+    Returns the year from ``x``.
+
+.. spark:function:: unix_timestamp() -> integer
+
+    Returns the current UNIX timestamp in seconds.
+
+.. spark:function:: unix_timestamp(string) -> integer
+
+    Returns the UNIX timestamp of time specified by ``string``. By default
+    assume the format ``yyyy-MM-dd HH:mm:ss``.
+
+.. spark:function:: unix_timestamp(string, format) -> integer
+
+    Returns the UNIX timestamp of time specified by ``string`` using the
+    format described in the ``format`` string.
```

Comment:
Would you document the 'format' or link to an existing documentation (perhaps, in Spark)?

Reply:
Would you also document the behavior in case 'time' string doesn't match 'format' or 'format' is not a valid format string.